moto/__init__.py

```diff
@@ -1,5 +1,6 @@
 import importlib
 import sys
+from contextlib import ContextDecorator
```

Comment:
First available in Python 3.2 - what's the minimum supported moto version? Judging from setup.py, it's 3.5, but I'm not sure if there's an effort to remain further backwards compatible.

Reply:
It's 3.5 indeed - although that's been EOL for a while now, so we'll probably drop that soon. There's no need to make things backwards compatible, so using this import is fine